repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -128,9 +128,7 @@ class ReposeJettyServer(val clusterId: String,
           }
         }
 
-        Option(ssl.isTlsRenegotiationAllowed).foreach { tls =>
-          cf.setRenegotiationAllowed(tls)
-        }
+        cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse[java.lang.Boolean](false))
```

Comment:
Sure.  I thought the line looked ugly that way, but why not.

Of note, I tried:
```
Option(ssl.isTlsRenegotiationAllowed).getOrElse(java.lang.Boolean.FALSE)
```
but the build failed with:
```
[ERROR]  found   : Any
[ERROR]  required: Boolean
[ERROR]         cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse(java.lang.Boolean.FALSE))
[ERROR]                                                                                   ^
[ERROR] one error found
```
which doesn't make a lot of sense to me.

I considered for a moment importing java.lang.Boolean to avoid writing out the whole thing twice, but I didn't want to risk messing something else up that depended on the scala boolean.